README.rst

```diff
@@ -83,17 +83,26 @@ Examples
    # You can weight over the dimensions the function is being applied
    # to by passing the argument ``weights=weight`` with a xr.DataArray
    # containing the dimension(s) being reduced.
```

Comment:
Made adding a bit of a 'story' about this example. e.g. it is common to weight latitude values with a cosine weights which is maximum at the equator and minimum at the poles. As we grow the docs and add notebooks this helps a bit.